### PR TITLE
Fix debian start for heartbeat/apache

### PR DESCRIPTION
**Pull request by dhoffend**
* Added Debian Configuration File
* Added fix for folder detection/creation/configtest for debian based default configurations. This fix also removes the hardcoded ubuntu workaround

### CODE REVIEW

**Comment by dmuhamedagic** [comment]
On Sat, Feb 13, 2016 at 05:42:28PM -0800, Daniel Hoffend wrote:
> * Added Debian Configuration File
> * Added fix for folder detection/creation/configtest for debian based default configurations.

Great!

Can you please just prefix the commit messages with:

	Medium: apache:

> This fix also removes the hardcoded ubuntu workaround

First looked at the code. Please ignore the comment about it, if
the other part of your patch obsoletes this part.

**Comment by dhoffend** [comment]
Okay, I'll rollback, fix the commit prefixes and replace the bashims test with a find+grep combination. This way I can keep support of multi-instance configuration stuff and still don't require a bash.

**Comment by dhoffend** [comment]
Okay I pushed a new tested version into this pull request with updated commit message. I hope that the explanations in the commit message are clear enough.

File: heartbeat/apache [hunk 1]
```diff
@@ -173,6 +176,24 @@ validate_default_suse_config() {
...
+# Debian's Default configuration uses a lock directory /var/lock/apache2
+# which is only generated using the lsb init script issues configtest. To
+# ensure these default directories are present it's useful to run a configtest
+# prior to the resource startup which will create the needed directories
+#
+# To support multiple apache instances the debian scripts and configs
+# obey apache2/envvars. (copy /etc/apache2 -> /etc/apache2-instance)
+# adjust (SUFFIX) envvars and set OCF_RESKEY_envfiles
+validate_default_debian_config() {
+	if find /etc/apache2* -name apache2.conf | grep -q $CONFIGFILE
```
Comment (guessi) [inline heartbeat/apache]: for preventing word-splitting issue, better to quote `"$CONFIGFILE"`

Comment (dhoffend) [inline heartbeat/apache]: Sure. I'll fix that and reupload the changes again

**Comment by krig** [comment]
Looks good to me. Merging.

Thanks!

**Pull request merged**